lib/index.js

```diff
@@ -1003,6 +1041,52 @@ MapboxGeocoder.prototype = {
     }
   },
 
+  /**
+   * Handle the placement of a result marking the selected result
+   * @private
+   * @param {Object[]} results the top results to display on the map
+   * @returns {MapboxGeocoder} this
+   */
+  _handleResultMarkers: function(results){
+    // clean up any old marker that might be present
+    if (!this._map){
+      return;
+    }
+    this._removeResultMarkers();
+    var defaultMarkerOptions = {
+      color: '#4668F2'
```

Comment:
Is this default a standard?

Reply:
This is the same color used as the original `_handleMarker` method, just copied the color from there pretty much